contents/docs/web-analytics/web-vitals.mdx

```diff
@@ -154,6 +154,39 @@ Metrics are made available when the [web vitals library](https://github.com/Goog
 You can look at our Web vitals capturing code in [web-vitals/index.ts](https://github.com/PostHog/posthog-js/blob/main/src/extensions/web-vitals/index.ts).
 
 
+### Sampling `$web_vitals` events
+
+If you're concerned about the cost of `$web_vitals` events, you can enable [sampling](/docs/libraries/js#sampling-events) for `$web_vitals` events - this will reduce the amount of events you send at the cost of some accuracy.
```

Comment:
```suggestion
If you're concerned about the cost of `$web_vitals` events, you can enable [sampling](/docs/libraries/js#sampling-events) for `$web_vitals` events. This will reduce the amount of events you send at the cost of some accuracy.
```